certified-connectors/Projectum PresentIt/readme.md

```diff
@@ -0,0 +1,34 @@
+﻿
+## Projectum Present It
+Present It allows you to use your PowerPoint and Word templates to dynamically fill in your business data.
+By a single click on a button, you can skip all the manual copy/paste of data, and just have Present It fill it out for you.
+https://projectum.com/
+
+
+## Pre-requisites
+An API key is required.
+Sign-up for a trial Key here: https://forms.office.com/Pages/ResponsePage.aspx?id=Kns17vkbpkK6q5dy2FsowQ7i3LhuXXpOhvMyBVDJFPNUNEE5OUo2RzUxVDM5OVIxME5CRzRTQUcwQy4u
+
+
+## API documentation
+https://docs.microsoft.com/en-us/connectors/projectum/
+
+
+## Supported Operations
+Action: Generate Word document
+Description: Generates a Word document based on JSON data.
+
+Action: Generate PowerPoint document
+Description: Generates a PowerPoint document based on JSON data.
+
+https://docs.microsoft.com/en-us/connectors/projectum/
+
+
+## How to get credentials
+To sign-up for a trial, go here: https://forms.office.com/Pages/ResponsePage.aspx?id=Kns17vkbpkK6q5dy2FsowQ7i3LhuXXpOhvMyBVDJFPNUNEE5OUo2RzUxVDM5OVIxME5CRzRTQUcwQy4u
+You will recieve an API key and a 'Get started' guide
+
+
+## Deployment instructions
+Please use [these instructions](https://docs.microsoft.com/en-us/connectors/custom-connectors/paconn-cli) to deploy this connector as custom connector in Microsoft Flow and PowerApps
```

Comment:
Please change this to say "...as a custom connector in Microsoft Power Automate and Power Apps" Thanks!